setup.py

```diff
@@ -13,7 +13,7 @@
     packages=[
         "dataflux_core", "dataflux_pytorch", "dataflux_pytorch.lightning",
         "dataflux_pytorch.multipart_upload", "demo.lightning.text_based",
-        "demo.lightning.checkpoint.multinode"
+        "demo.lightning.checkpoint.multinode","demo.image_segmentation"
```

Comment:
Maybe not for this PR but I'm not sure the demo code actually needs to be included here. Do things break if we remove it?

Reply:
Can we just use `-m`? In general we should be using `-m`

Reply:
I think we can do that. Along with Kokoro we have to update some of the readme too. 
I'll create a new PR to remove demo from setup.py

Reply:
Resolving the comment for now.